apps/openchallenges/mcp-server/src/main/resources/application.yml

```diff
@@ -1,3 +1,7 @@
+openchallenges-mcp-server:
```

Comment:
@tschaffter to keep config data simplify, I didn't use nest property for this line.